packages/datadog-plugin-jest/src/index.js

```diff
@@ -41,6 +41,14 @@ class JestPlugin extends Plugin {
     this.testEnvironmentMetadata = getTestEnvironmentMetadata('jest', this.config)
     this.codeOwnersEntries = getCodeOwnersFileEntries()
 
+    this.addSub('ci:jest:test:code-coverage', (coverageFiles) => {
+      if (!this.config.isAgentlessEnabled || !this.config.isITREnabled) {
```

Comment:
ITR?

Reply:
intelligent test runner

Reply:
Could we use some terminology that is not an unknown acronym to make this easier to use?

Reply:
yeah that makes sense

Reply:
done